tests/test_lazy_imports.py

```
@@ -0,0 +1,151 @@
+We run each scenario in a fresh subprocess so module-import side effects from
+the rest of the test suite cannot mask a regression.
+"""
+
+import subprocess
+import sys
+import textwrap
+
+
+def _run(script: str) -> str:
+    """Execute ``script`` in a fresh Python interpreter and return stdout."""
+    result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)